depfinder/main.py

```diff
@@ -70,11 +70,24 @@
     Loader=yaml.SafeLoader,
 )
 
+with open('depfinder/pkg_data/name_mapping.yaml', 'r') as f:
+   mapping_list = yaml.load(f, Loader= yaml.SafeLoader)
```

Comment:
You might consider trying to pull this from the website directly first, and then loading it from file if you can't get anything from GitHub.

Reply:
done